### PR TITLE
Removed Props object as it wasnt used anywhere in this project and I don't anticipate it being useful

### PR DESCRIPTION
**Pull request by CodeByAlex**
## What I did

If we ever need to add the feature back in we can but this is currently cluttering the configuration when it doesn't need to be.

If anyone disagrees with this, could you provide an example of when this might be useful so we can focus the feature on solving a problem?

### CODE REVIEW

File: app/aem/src/client/preview/render.ts [hunk 1]
```diff
@@ -16,15 +16,13 @@ const rootElement = document.getElementById('root');
...
+const createRuntime = (wcmmode, content, resourceLoaderPath, compLoader, components, models) => {
   const resolver = new ResourceResolver(content || {}, compLoader, components);
   const runtime = new Runtime();
   runtime.setGlobal({
     models,
     wcmmode,
-    component: {
-      properties: props
-    },
+    component: {},
```
Comment (tripodsan) [inline app/aem/src/client/preview/render.ts]: can you leave an empty object?

**Comment by CodeByAlex** [comment]
> [L6](https://github.com/adobe/aem-guides-wknd/blob/f56cdd79fa4b8ee2451d34b8d3f2120bb94800fb/ui.apps/src/main/content/jcr_root/apps/wknd/components/content/accordion/.content.xml#L6)
> 
> but they should be loaded automatically from the `.content.xml` and not needed to be defined in the story. so, ok to remove them for now
Exactly👍

**Comment by CodeByAlex** [comment]
> yes, we need them to resolve the super type of the components. eg:
> https://github.com/adobe/aem-guides-wknd/blob/f56cdd79fa4b8ee2451d34b8d3f2120bb94800fb/ui.apps/src/main/content/jcr_root/apps/wknd/components/content/accordion/.content.xml#L6

Yes, of course but we can derive this data from the content

**Comment by CodeByAlex** [comment]
> My initial idea was, to derive the story title automatically from the `jcr:title` of the component. But I think the story titles want to convey more information than just what's in the component title. (eg "Text with Knobs").

Yeah I think these are better off separate

File: app/aem/src/client/preview/render.ts (hunk 1 above)
Comment (CodeByAlex) [reply app/aem/src/client/preview/render.ts]: I can do that 👌

**Pull request merged**